Objective-J/Bootstrap.js

```diff
@@ -45,6 +45,23 @@ if (DOMBaseElementsCount > 0)
         pageURL = new CFURL(DOMBaseElementHref, pageURL);
 }
 
+// Set compiler flags
+
+if (OBJJ_COMPILER_FLAGS)
```

Comment:
This generated a Uncaught ReferenceError: OBJJ_COMPILER_FLAGS is not defined on Chrome. 

```js
typeof OBJJ_COMPILER_FLAGS !== 'undefined'
```

would be better.